mkosi/qemu.py

```diff
@@ -788,11 +788,19 @@ def finalize_state(config: Config, cid: int) -> Iterator[None]:
             p.unlink(missing_ok=True)
 
 
-def finalize_kernel_command_line_extra(config: Config) -> list[str]:
+def finalize_kernel_command_line(config: Config, kernel: Optional[Path], firmware: QemuFirmware) -> list[str]:
```

Comment:
That's what ruff is complaining about.
```suggestion
def finalize_kernel_command_line(
    config: Config, 
    kernel: Optional[Path], 
    firmware: QemuFirmware,
) -> list[str]:
```